.travis.yml

```diff
@@ -2,17 +2,14 @@ language: ruby
 cache: bundler
 sudo: false
 rvm:
-  - 2.2
-  - 2.3
-  - 2.4
```

Comment:
**[SHOULD]** Ruby 2.4 is maintained yet.

EOL is `2020-03-31`
https://www.ruby-lang.org/en/downloads/branches/

So I think this shouldn't be erased

Reply:
I see.
OK, I will revert removing old ruby CI.